peps/pep-0736.rst

```diff
@@ -3,7 +3,7 @@ Title: Shorthand syntax for keyword arguments at invocation
 Author: Joshua Bambrick <jbambrick@google.com>,
         Chris Angelico <rosuav@gmail.com>
 Sponsor: Guido van Rossum <guido@python.org>
-Discussions-To: https://discuss.python.org/t/syntactic-sugar-to-encourage-use-of-named-arguments/36217
+Discussions-To: https://discuss.python.org/t/syntactic-sugar-to-encourage-use-of-named-arguments/36217 https://discuss.python.org/t/pep-736-shorthand-syntax-for-keyword-arguments-at-invocation/43432
```

Comment:
You should remove the old link from this field and add both to Post-History below.

Reply:
Thanks!
Not sure what is meant by
> add both to Post-History below.

though.

Reply:
Discussions-To is just the most recent link.

Post-History (line 11) is a list of all discussions, including the most recent.

For example, see:

https://github.com/python/peps/blob/c4a5976ff3fe9aad81b12c0b6ed1b64d406cfa48/peps/pep-0712.rst#L5-L13

Reply:
Thanks both for the kind suggestions.
Apologies on my end for not looking up!